data

```diff
@@ -1 +1 @@
-Subproject commit bdc5311b6d7a22c445ddcc80143a5fc21368836b
+Subproject commit 28012f3c9d89a8987ad85026d4b7487f38276906
```

Comment:
@maxteufel Honestly I don't even know what I changed and don't know how to delete it, I haven't made any changes to the data repository.